cmd/bootstrap/cmd/google.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+// GoogleBucket ...
+type GoogleBucket struct {
+	Name string
+}
+
+// NewGoogleBucket ...
+func NewGoogleBucket(bucketName string) *GoogleBucket {
```

Comment:
👉  consider making these functions package private by renaming them to start with a lower case letter. e.g. newGoogleucket, newClient, getFiles etc.

Reply:
Thanks Vishal